src/Common/src/Interop/Linux/Interop.Data.cs

```diff
@@ -1,12 +1,11 @@
-﻿// Licensed to the .NET Foundation under one or more agreements.
+// Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-namespace System.Data.Common
+internal static partial class Interop
 {
-    internal static class ExternDll
+    internal static partial class Data
     {
         public const string Odbc32 = "libodbc.so.2";
     }
-}
-
+}
```

Comment:
Nit: some editors make sure files end with a newline when saving (it's an old Unix convention, which is why git highlights it). So if you remove them, they tend to come back and cause another diff. Generally better to leave them